### PR TITLE
Add Helm Charts for Identity server with analytics

### LINKED ISSUE #31: Add Helm Charts for Identity server with analytics
**Description:**
Helm charts should be created for the deployment of clustered Identity Server with analytics.

**Suggested Labels:**
Type/NewFeature

**Affected Product Version:**
WSO2 Identity server 5.5.x

### PR DESCRIPTION
**Pull request by ThilinaManamgoda**
Also contains a fix for the claim name in identity-server-deployment.yaml file.

### CODE REVIEW

File: helm/is-with-analytics/is-with-analytics-conf/values.yaml [hunk 1]
```diff
@@ -4,8 +4,8 @@ email: "<EMAIL>"
...
-sharedDeploymentLocationPath: "/data/deployment"
-sharedTentsLocationPath: "/data/tenants"
-analytics1DataLocationPath: "/data/data-1"
-analytics2DataLocationPath: "/data/data-2"
\ No newline at end of file
+serverIp: "<SERVER_IP>"
+sharedDeploymentLocationPath: "<NFS_LOCATION>"
+sharedTentsLocationPath: "<NFS_LOCATION>"
+analytics1DataLocationPath: "<NFS_LOCATION>"
+analytics2DataLocationPath: "<NFS_LOCATION>"
```
Comment (KavinduZoysa) [inline helm/is-with-analytics/is-with-analytics-conf/values.yaml]: shall we add a new line here?

Comment (msmshariq) [reply helm/is-with-analytics/is-with-analytics-conf/values.yaml]: +1

Comment (ThilinaManamgoda) [reply helm/is-with-analytics/is-with-analytics-conf/values.yaml]: Done

File: is-with-analytics/is/identity-server-deployment.yaml [hunk 2]
```diff
@@ -90,7 +90,7 @@ spec:
           name: identity-server-conf-event-publishers
       - name: shared-deployment-persistent-disk
         persistentVolumeClaim:
-          claimName: scalable-is-shared-deployment-volume-claim
+          claimName: is-shared-deployment-volume-claim
```
Comment (msmshariq) [inline is-with-analytics/is/identity-server-deployment.yaml]: shale we make this `identity-shared-deployment-volume-claim`, if not it looks like a boolean variable.

Comment (ThilinaManamgoda) [reply is-with-analytics/is/identity-server-deployment.yaml]: +1, we will make this change for next release.

Comment (msmshariq) [reply is-with-analytics/is/identity-server-deployment.yaml]: Yes, please create a issue to track this. We need to fix the across the repo.

**Pull request merged**